src/v3/src/components/AuthCoin/authenticatorConfiguration.ts

```diff
@@ -96,11 +96,6 @@ const AuthenticatorConfiguration: {
     customizable: true,
     description: 'Security Question',
   },
-  [AUTHENTICATOR_KEY.SMS]: {
-    icon: SMSIcon,
-    customizable: true,
-    description: 'SMS',
-  },
```

Comment:
Is this not used anymore?

Reply:
After some research SMS was never an authenticator Key, so this was never used. https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/RemediationConstants.js#L95-L111